backend/internal/controller/server/msg_handler.go

```diff
@@ -69,7 +69,8 @@ func (s server) createResponse(reader io.Reader) ([]byte, error) {
 		return nil, err
 	case adapter.MsgPosition:
 		log.Trace("X coord of the ball: ", message.X, " Y coord of the ball: ", message.Y)
-		return nil, nil
+		err := s.game.IncrementHeatmap(message.X, message.Y)
```

Comment:
In my opinion, it is redundant to create a new `error` variable here. Instead, you can invoke this function in the `return` statement.

Reply:
Good point, I will change it.